app/Services/ReportbackService.php

```diff
@@ -34,6 +35,38 @@ public function create($data)
     {
         $reportback = $this->reportbackRepository->create($data);
 
+        // POST reportback back to phoenix.
+        $body = [
+            'uid' => $reportback->drupal_id,
+            'nid' => $reportback->campaign_id,
+            'quantity' => $reportback->quantity,
+            'why_participated' => $reportback->why_participated,
+            'file_url' => $reportback['items'][0]['file_url'],
```

Comment:
Hmm, since this is an Eloquent model, can this be re-written `$reportback->items()->first()->file_url`? Or maybe these should be computed properties on the Reportback model (that gets each field from the first item)? It'd be nice to avoid working on things as plain old arrays whenever possible, and it feels like it could be a little dangerous just "reaching in" from here.